plugins/browser.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
Any idea where this config file came from and if it's necessary in source control?